examples/preprocessing/plot_eeg_csd.py

```diff
@@ -0,0 +1,94 @@
+"""
+=====================================================
+Transform EEG data using current source density (CSD)
+=====================================================
+
+This script shows an example of how to use CSD [1]_ [2]_ [3]_.
+CSD takes the spatial Laplacian of the sensor signal (derivative in both
+x and y). It does what a planar gradiometer does in MEG. Computing these
```

Comment:
(nitpick) isn't it more accurate to say that CSD does what a *pair of co-located* planar gradiometers does in MEG?

Reply:
Each individual MEG gradiometer actually computes its own spatial gradient. The pair of gradiometers at each location compute gradients in orthogonal directions.

Reply:
right, so if CSD does derivative in both x and y directions, isn't the result of CSD like the result of combined grad pairs?  (If I'm wrong no need to explain, just say "you're wrong" and I'll go read some papers) :)

Reply:
We combine grad pairs by doing a norm, which will be quite different from computing the spatial derivative along two dimensions

Reply:
I agree that this sentence (i may have suggested) is misleading/wrong. planar grads get combined with RMS while here laplacian is summing the second order derivatives in both directions (divergence of the gradient).